manifests/g/G3G4X5X6/ultimate-cube/6.6.11/G3G4X5X6.ultimate-cube.locale.en-US.yaml

```
@@ -0,0 +1,41 @@
+# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 $debug=QUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: G3G4X5X6.ultimate-cube
+PackageVersion: 6.6.11
+PackageLocale: en-US
+Publisher: g3g4x5x6
```

```suggestion
Publisher: G3G4X5X6
```
This must match Control Panel.